.github/workflows/merged.yml

```diff
@@ -84,9 +84,7 @@ jobs:
           # import certificates from secrets
           echo -n "${{env.AWS_SECRET_DISTRIBUTION_P12_ENCODED}}" | base64 --decode -o $CERTIFICATE_PATH
           echo -n "${{env.AWS_SECRET_AUTH_KEY_P8_ENCODED}}" | base64 --decode -o $APIKEY_PATH
-
-          bundle exec fastlane run add_badge alpha:true
```

Comment:
Are we not fussed about adding the badge anymore? We should probably mention this in the pull request description.

If it's because we couldn't get it working in the time, can we raise a follow up ticket to investigate and reference that in the description too?

Reply:
@Oliver-Binns have updated the PR description to reflect we have removed it for now, and here is the ticket to investigate how to add it back in 
https://govukverify.atlassian.net/browse/DCMAW-6996